jina/peapods/pods/k8s.py

```diff
@@ -304,3 +235,14 @@ def _get_base_executor_version(self):
             return jina.__version__
         else:
             return 'master'
+
+    def _create_node(self, suffix):
+        name = f'{self.name}_{suffix}'
```

Comment:
The underscore is part of the suffix I think

Reply:
thanks - fixed